core/shared/src/main/scala/zio/stm/TMap.scala

```diff
@@ -158,6 +158,12 @@ final class TMap[K, V] private (
   def toList: STM[Nothing, List[(K, V)]] =
     fold(List.empty[(K, V)])((acc, kv) => kv :: acc)
 
+  /**
+   * Collects all bindings into a map.
+   */
+  def toMap: STM[Nothing, Map[K, V]] =
+    fold(Map.empty[K, V])((acc, kv) => acc + (kv._1 -> kv._2))
```

Comment:
Looks like you can simplify:
```suggestion
    fold(Map.empty[K, V])(_ + _)
```

Reply:
thanks!